services/api-server/setup.cfg

```diff
@@ -1,8 +1,11 @@
 [bumpversion]
-current_version = 0.3.0
+current_version = 0.3.1
 commit = True
 message = services/api-server version: {current_version} → {new_version}
 tag = False
 commit_args = --no-verify
 
 [bumpversion:file:VERSION]
+
+[tool:pytest]
+asyncio_mode = auto
```

Comment:
Question: For this to work, does it mean I have to run `pip install .` or will it stil work if I run `make install-dev`?